src/parser.mly

```diff
@@ -96,6 +97,14 @@ args:
     expr            { [$1] }
   | expr COMMA args { $1::$3 }
 
+elems_opt:
+    /*nothing*/  { [] }
+  | elems        { $1 }
+
+elems:
+    expr             {[$1]}
```

Comment:
Hypernit: `{ [$1] }`

Reply:
Also isn't `elems_opt` and `elems` the same as the `args_opt` and `args` rules?

Reply:
yeah, they are, but since they're used in diff places I deliberately redefined it. should we name it something general so it can be used in both locations?

Reply:
That would be clearer I think